### PR TITLE
Assignment4

### PR DESCRIPTION
**Pull request by AlejandroGil14**

### CODE REVIEW

File: task06.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Copia de Task06.ipynb
```
Comment (dgarijo) [inline task06.py:2]: Remove this file. Leave only the ones in your folder

File: AlejandrGil14-22c054/task06 [hunk 2]
```diff
@@ -0,0 +1,95 @@
+**Task 06: Modifying RDF(s)**
+"""
+
```
Comment (dgarijo) [inline AlejandrGil14-22c054/task06:3]: File has the incorrect extension

File: AlejandrGil14-22c054/task06 [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.3: Create a new individual of Researcher named "Jane Smith"**"""
+
+# TO DO
+fullName = Literal("Jane Smith")
+EX = Namespace("http://example.org/")
+janeURI = ns.JaneSmith
+resource = (janeURI ,ns.Researcher, fullName)
```
Comment (dgarijo) [inline AlejandrGil14-22c054/task06:48]: The property used is incorrect. You are adding a class as a property!

File: AlejandrGil14-22c054/task06 [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+print("----------------------------------------")
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.4: Add to the individual JaneSmith the email address, fullName, given and family names**"""
+
+# TO DO
+# Visualize the results
+vcard = Namespace("http://www.w3.org/2001/vcard-rdf/3.0#")
+email=(janeURI, vcard.EMAIL, Literal('jsmith@example.org'))
```
Comment (dgarijo) [inline AlejandrGil14-22c054/task06:61]: We have repeatedly shown in class the importance of adding xsd types in literals. Please follow those guidelines.

File: AlejandrGil14-22c054/task06 [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+JohnUri = ns.JohnSmith
+UPM_uri = EX.Upm
+UPM = (UPM_uri, RDF.type, vcard.University)
+relacion = (JohnUri, vcard.WorksAt, UPM_uri)
```
Comment (dgarijo) [inline AlejandrGil14-22c054/task06:78]: vcard does not have a `WorksAt` property

**Review by dgarijo — CHANGES_REQUESTED** [review]
Please see comments about errors and missing things

**Comment by anaigmo** [comment]
Entregado pero faltan ejercicios, no se admiten más PR

**Pull request merged**